frontends/concrete-python/concrete/fhe/mlir/processors/assign_bit_widths.py

```diff
@@ -88,13 +88,38 @@ def max_encrypted_bitwidth_node(node: Node):
         return normal_p + 1
 
     if name in MULTIPLY and all(value.is_encrypted for value in node.inputs):
-        return normal_p + 1
+        # For operations that use multiply, an additional bit
```

Comment:
Why does multiply change here when the PR is about matmul ?
The previous version assumes multiply is single precision (i.e. input/output will be the same after this pass).
This is still the case since
`TLU_WITHOUT_PRECISION_CHANGE = CHUNKED_COMPARISON | MAX_POOLS | MULTIPLY`

Reply:
also I took "MULTIPLY" to mean "the set of operations that use enc x enc multiplication"